src/main/java/li/cil/oc2/common/bus/device/rpc/item/SoundCardItemDevice.java

```diff
@@ -21,7 +22,7 @@
 import java.util.function.Supplier;
 
 public final class SoundCardItemDevice extends AbstractItemRPCDevice {
-    private static final int COOLDOWN_IN_TICKS = TickUtils.toTicks(Duration.ofSeconds(2));
+    private static final int COOLDOWN_IN_TICKS = TickUtils.toTicks(Duration.ofSeconds(Config.soundCardCoolDownSeconds));
```

Comment:
This should be a non-static field now, I think.

While the config itself probably never changes in a meaningful way, there's the *possibility* that this class gets loaded before the config was initialized.

Reply:
it's been a while since i've dealt with java - would final still fit?

Reply:
Yes, field can still be final (it's like readonly in C# if that helps).